token-lending/program/src/state/obligation.rs

```diff
@@ -116,12 +116,16 @@ impl Obligation {
         &self,
         liquidity: &ObligationLiquidity,
     ) -> Result<Decimal, ProgramError> {
-        let max_liquidation_value = self
+        let mut max_liquidation_value = self
             .borrowed_value
             .try_mul(Rate::from_percent(LIQUIDATION_CLOSE_FACTOR))?
             .min(liquidity.market_value)
             .min(Decimal::from(MAX_LIQUIDATABLE_VALUE_AT_ONCE));
 
+        if self.deposited_value < Decimal::from(1u64) {
```

Comment:
Been awhile since i looked at this code but `Decimal::from(1u64)` is the smallest non-zero amount right? So is the logic here to allow `max_liquidation_value = self.borrowed_value.min(liquidity.market_value)` if the obligation deposit value is zero?

Reply:
need to look at this again but deposited value is in usd so this is just saying that if it's <$1 then you can fully liquidated the account